core/src/main/java/io/questdb/cutlass/http/processors/RejectProcessor.java

```diff
@@ -48,7 +48,7 @@ default void onPartBegin(HttpRequestHeader partHeader) {
     default void onPartEnd() {
     }
 
-    HttpRequestProcessor rejectRequest(int code, CharSequence userMessage, CharSequence cookieName, CharSequence cookieValue, byte authenticationType);
+    HttpRequestProcessor rejectRequest(int code, CharSequence userMessage, CharSequence cookieName, CharSequence cookieValue, byte authenticationType, boolean shutdownWrite);
```

Comment:
nit: line is too long

Reply:
RejectProcessor has been refactored, this method has gone